substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -4072,3 +4072,13 @@ impl<T: Config> sp_staking::OnStakingUpdate<T::AccountId, BalanceOf<T>> for Pall
 		}
 	}
 }
+
+#[pallet::view_functions]
```

Comment:
Have you tried to trigger this from anywhere now? 

I can provide some pointers on how to do it manually: 

* The `kitchensink` runtime should have a new runtime API that will expose this. 
* You can manually hash the name and signature of this function (as per the view fn PR) to get the `QueryId`
* You can use any of our developer consoles that can trigger a runtime API to call into the runtime API, and give the right query
  * https://dev.papi.how/
  * https://polkadot.js.org/apps/